actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsBackgroundActor.java

```diff
@@ -57,7 +58,9 @@ public void onReceive(Request request) throws Throwable {
   @SuppressWarnings("unchecked")
   public void courseProgressMetricsData(Request actorMessage) {
 
-    ProjectLogger.log("CourseMetricsActor-courseProgressMetrics called");
+    ProjectLogger.log(
+        "CourseMetricsBackgroundActor:courseProgressMetricsData courseProgressMetrics called",
```

Comment:
From:
"CourseMetricsBackgroundActor:courseProgressMetricsData courseProgressMetrics called"

To:
"CourseMetricsBackgroundActor: courseProgressMetricsData called."